plugins/editor/class.editor.plugin.php

```diff
@@ -1448,7 +1469,51 @@ public function postingSettings_formatSpecificFormItems_handler(
         $form->setValue('Plugins.editor.ForceWysiwyg', c('Plugins.editor.ForceWysiwyg'));
         $formToggle = $form->toggle('Plugins.editor.ForceWysiwyg', $forceWysiwygLabel, [], $label);
 
-        $additionalFormItemHTML .= "<li class='form-group'>$formToggle</li>";
+        $additionalFormItemHTML .= "<div class='form-group forceWysiwyg'>$formToggle</div>";
+
+        //Image Upload form items
+        $imageUploadLimitLabel = t('ImageUploadLimits.Notes1', 'Enable Image Upload Limit');
+        $ImageUploadDesc = t('ImageUploadLimits.Notes2', 'Add limits to image upload dimensions in discussions and comments.');
```

Comment:
Did you add these new translations to the locales repo?

Reply:
Yes.